src/frontend/js/media-management/library.tsx

```diff
@@ -40,14 +41,17 @@ export default function Library({
   expertMode,
   allowedMediaTypes,
   selectionMode,
+  onlyImage,
   selectMedia,
 }: Props) {
   // The directory path contains the current directory and all its parents
   const [directoryPath, setDirectoryPath] = useState<Directory[]>([]);
   // The current directory is the last element of the directory path
   const directory = directoryPath[directoryPath.length - 1];
   // The directory content contains all subdirectories and files of the current directory
-  const [directoryContent, setDirectoryContent] = useState<MediaLibraryEntry[]>([]);
+  const [directoryContent, setDirectoryContent] = useState<MediaLibraryEntry[]>(
+    []
+  );
```

Comment:
I'm not sure if I really find this more readable than before... Maybe you can configure prettier with a slightly longer max line length? E.g. 88 characters like black or 100 chars?

Reply:
Yeah, Will do so :)